python/cuml/test/test_coordinate_descent.py

```diff
@@ -38,7 +38,7 @@ def stress_param(*args, **kwargs):
 
 
 @pytest.mark.parametrize('datatype', [np.float32, np.float64])
-@pytest.mark.parametrize('X_type', ['dataframe', 'ndarray'])
+@pytest.mark.parametrize('X_type', ['ndarray'])
```

Comment:
Yes, since the input_utils are now tested running this twice is not testing anything new since the algorithmic part ends up receiving the same device data, just running the algorithms twice with the exact same data.